feature-add-meal/build.gradle.kts

```diff
@@ -0,0 +1,99 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Should we convert to a classic gradle?

Reply:
I think so, because then you can apply from common.gradle and means you don't have to duplicate a lot of things